### DISCUSSION TITLE
Content Editor Components - Module Dropzone Card

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#2893] [Office hours] [answered]
Hi again! I wanted to continue to chat about the Drag & Drop components, but thought it would make more sense to divide up this work into a couple different conversations to help focus future office hours sessions on specific areas of the drag & drop pattern. I'm working on what i'm calling the module drop zone card (name suggestions welcome). 

All drop zone cards will be slightly different in what actions need to be included (3 need HTML buttons, the rest don't) as well as what the body of the card will contain. All require delete, copy, and move/drag. 

This one is basically a Pastified version of current MC: 
![Screenshot 2022-12-14 at 12 08 03 PM](https://user-images.githubusercontent.com/87141551/207690773-2f338a74-413c-472e-a904-553b2b910cfe.png)

This is a Pastified Lightning drop zone component (ish):
![Screenshot 2022-12-14 at 12 07 45 PM](https://user-images.githubusercontent.com/87141551/207690657-33da89e1-5059-4116-a63d-9f7362f5b562.png)

Looking forward to chatting during office hours after the holidays, thanks!

**Comment by nkrantz** [comment, marked as answer]
Hey @ceyerkes, thanks for coming to Office Hours.

[Here's the recording from yesterday](https://www.loom.com/share/956d9891613f4634af392f75f71dc478) (find the password in #help-design-system slack).

This is definitely a unique type of flow, so some of these UX patterns will have to be learned. Generally we don't love controls that appear on hover, but it makes sense here, especially because they also show up in the focus/selected state. Also consider whether any of your smaller components/modules can exist without the same type of container, or whether that might be more confusing for users. 

All in all, your work is coming along wonderfully, we're excited to see how it progresses! Feel free to keep using these threads for more questions/updates.

**Reply by ceyerkes** [reply]
Thanks Nora, appreciate all the feedback as usual!

**Comment by ceyerkes** [comment]
[Drop Zone Card update via Loom](https://www.loom.com/share/ada352e19c2647c7a57bea480fecaee1) as requested!

I ran out of time, but i'm wondering if i can use an icon-only menu dropdown for the modules that are dropped inside of columns to save space, kind of like how MC handles this now:

![Screenshot 2023-01-26 at 10 19 06 AM](https://user-images.githubusercontent.com/87141551/214904587-0946dffa-fb87-430c-a6e2-a85dfed758f9.png)

I can bring this back to office hours, or we can chat async, whatever works!
